typescript-selenium-webdriver-sample/tests/webdriver-factory.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as webdriver from 'selenium-webdriver';
+import * as chrome from 'selenium-webdriver/chrome';
+import * as firefox from 'selenium-webdriver/firefox';
+import * as path from 'path';
+
+export function createWebdriverFromEnvironmentVariableSettings(): webdriver.ThenableWebDriver {
+    // Selenium WebDriver implementations generally require that you use a version of the webdriver
+    // that exactly matches the version of the browser it is driving.
+    //
+    // In our Azure Pipelines build agents, using the pre-installed versions of the webdrivers that
```

Comment:
```
    // In our Azure Pipelines build agents, using the pre-installed versions of the webdrivers that
    // match the pre-installed versions of the browsers.
```
not sure if I'm missing something here but the phrasing seems a little off to me

Reply:
agreed, fixed